packages/js/data/changelog/fix-export-button-error

```diff
@@ -0,0 +1,4 @@
+Significance: fix
+Type: minor
+
+Fix 'Cannot read properties of undefined' error when clicking Export button on Analytic pages.
```

Comment:
Should this have been added to the changelog?

Reply:
I'm not really sure how we should handle changelogs for consumed packages. It's possible that a package might have something fixed for an external consumer, for instance. Let's start a discussion.